examples/traffic_histories_vehicle_replacement.py

```diff
@@ -183,6 +198,7 @@ def custom_filter(vehs: Iterable[VehicleWindow]) -> Iterable[VehicleWindow]:
             ego_missions = {}
             sample = set()
 
+            print(scenario.traffic_history.dataset_source)
```

Comment:
We should be using logging rather than prints generally, especially since we already have logger being used.

Reply:
This was actually a debug statement I forgot to remove. Thanks for catching!